Adjust documentation (#213)
clp-research/slurk: docs/slurk_layouts.rst

      }

- ``"script"``: Provides the ability to inject a script plugin into the chat. It consists of another dictionary, which
  maps different triggers to scripts or a list of scripts. Scripts can either be a predefined one or can be passed by
  link. Example:

    .. code-block:: json

      {
        "incoming-text": "display-text",
        "submit-message": "send-message",
        "print-history": "plain-history"
      }

Triggers
~~~~~~~~

A full list of available keys is shown in the :ref:`slurk_api`. There are some predefined scripts
which can be used. Also, several variables may be defined in a trigger:

``"incoming-text"``
-------------------
Displays text messages as they arrive

Variables:
  - ``data.user``: The user who has sent the message
  - ``data.timestamp``: The timestamp of the message
  - ``data.private``: A boolean value if this was a direct message or visible to the room
  - ``data.message``: The message string of the sent message if any
  - ``data.html``: Tag if the message is marked as HTML
Examples:
  - ``"display-text"``: Prints as plain text
  - ``"markdown"``: Prints as formatted markdown if tagged as html

``"incoming-image"``
--------------------

> Displays images in the chat area they arrive

Reviewer: ```suggestion
Displays images in the chat area as they arrive
```

Variables:
  - ``data.user``: The user who has sent the message
  - ``data.timestamp``: The timestamp of the message

>   - ``data.private``: A boolean value if this was a direct message or visible to the room

Reviewer: ```suggestion
  - ``data.private``: A boolean value indicating whether this was a direct message or visible to the room
```

  - ``data.image``: The image URL
  - ``data.width``: The width of the sent image
  - ``data.height``: The height of the sent image
Examples:

>   - ``"display-image"``: Displays ss a simple image

Reviewer: ```suggestion
  - ``"display-image"``: Displays a simple image
```
🐍

Reviewer 2: 🐍

  - ``current_user``: The user who just hit RETURN
  - ``current_timestamp``: The current timestamp
Examples:
  - ``"send-message"``: Sends plain text and commands depending on the entered text

``"print-history"``
-------------------
Shows previous messages in the chat area after joining a room

Variables:
  Not all variables in ``element.data`` may be defined

  - ``element.event``: Type of the event
  - ``element.user``: The user who sent the event
  - ``element.timestamp``: The timestamp of the event
  - ``element.data.message``: The message of the text event
  - ``element.data.url``: The URL of an image
  - ``element.data.width``: The width of an image
  - ``element.data.height``: The height of an image
  - ``element.receiver``: The receiver, if it was a private message
  - ``element.command``: The command which was executed
Examples:
  - ``"plain-history"``: As plain text and images
  - ``"markdown-history"``: Formatted as markdown if tagged as html
  - ``"attribute-history"``: Applies previous changes to the layout

``"typing-users"``
------------------
Called when state of currently typing users is changed

Variables:

>   - ``users``: A map of currently typing users, with its id as the key

Reviewer 2: with whose id as the key?

Called when state of currently typing users is changed

Variables:

>   - ``users``: A map of currently typing users, with whose id as the key

Reviewer: I think jana really did not understand who was meant and didn't intend to replace its by whose.
```suggestion
  - ``users``: A map of currently typing users, with their IDs as keys
```
